python/ray/tests/test_multi_tenancy.py

```diff
@@ -236,5 +237,33 @@ def foo():
     wait_for_condition(lambda: len(get_workers()) == before)
 
 
+def test_not_killing_workers_that_own_objects(shutdown_only):
+    # Set the small interval for worker capping
+    # so that we can easily trigger it.
+    ray.init(num_cpus=2, _system_config={"kill_idle_workers_interval_ms": 10})
+
+    # Create a nested tasks to start 10 workers each of which owns an object.
```

Comment:
Shouldn't it be 8 rather than 10?

Reply:
oh yeah good catch. It was originally 10 haha.